user-manual/ingest/manual-normalization.rst

```diff
@@ -4,261 +4,357 @@
-   Processing Location defined for your pipeline.
+The term "manual normalization" is used to refer to two different
+workflows in Archivematica:
 
-4. Choose which files you'd like to normalize for preservation and/or access.
+#. Adding normalized files to a transfer before ingesting it into Archivematica
+#. Adding normalized files to the SIP during the Normalize microservice
 
-5. Normalize locally using your preferred tools.
+For both options, the user has the choice to add PREMIS event detail metadata
+that describes the manual normalization events.
 
-6. Once normalization is completed, place your preservation and/or access copies
-   in a directory structured exactly like so:
+Manually normalized derivatives can be added to Standard transfers. Other
+transfer types are not supported.
 
-.. code:: bash
+*On this page*
+
+* :ref:`Workflow 1: Manual normalization prior to transfer to Archivematica <man-norm-before>`
+* :ref:`Workflow 2: Manual normalization during Ingest <man-norm-ingest>`
+* :ref:`Adding PREMIS eventDetail for manual normalization <man-norm-premis-detail>`
+* :ref:`Normalizing files with the same name <man-norm-same-name>`
+
+.. _man-norm-before:
 
-        /topDirectory
+Workflow 1: Manual normalization prior to transfer to Archivematica
+-------------------------------------------------------------------
 
-            /manualNormalization
+This workflow assumes that users have created access and/or preservation copies
+of their files before the material is transferred into Archivematica.
 
-                /access
+For example, during a digitization project, high-resolution original (master)
+images as well as lower-resolution access copies may be created. It would be
+redundant of Archivematica to duplicate this normalization work. By adding a
+folder called ``manualNormalization`` to the transfer prior to processing, the
+Archivematica can recognize that normalization work has already been done so
+that Archivematica does not replicate the effort.
 
-                    file1.jpg
+.. note::
+   This workflow requires a one-to-one relationship an original object and its
+   access or preservation derivative. Creating multiple access or preservation
+   derivatives that relate to a single original object is not supported.
+
+#. Create your files - the original (master) copies and the manually normalized
+   derivatives. Note that the filenames must match in order for Archivematica to
+   recognize the links between the original file and its derivatives.
+
+   .. code:: bash
+
+      to-be-preserved/
+      ├── hooded-pitcher-plant.gif
+      ├── hooded-pitcher-plant.png
+      ├── hooded-pitcher-plant.tif
+      ├── ornamental-onion.gif
+      ├── ornamental-onion.png
+      ├── ornamental-onion.tif
+      ├── pepperbox-poppy.gif
+      ├── pepperbox-poppy.png
+      └── pepperbox-poppy.tif
+
+   In this example, there are three copies of each file - a GIF, a PNG, and a
+   TIFF.
```

Comment:
It might be clarifying here to give a hint of where this is going - "The PNG files will be used as the original files, the TIF files as preservation derivatives and the GIF files as access derivatives." Or similar.

Reply:
Or alternatively, you could start with an intro that kind of outlines the scenario: "A user has a set of files that uses PNG as the original format, TIF as the preservation derivative and GIF as the access derivative..."

Reply:
Good call!